### DISCUSSION TITLE
How can i refresh packages that came orginally from nuget.org

### DISCUSSION THREAD

**Discussion by janusqa** [dotnet/aspnetcore#55665] [Q&A]
I have a .net 8 blazor web app.
I have created a nuget.config file for it and change where some packages previously fetched from nuget.org, I now want to fetch from a local directory. However when using dotnet watch the packages used seem to what was originally downloaded from nuget.org and not from the new source which is a local directory.  If i were to build and publish only then are the packages fetched from the local directory (as its like starting from scratch).  How can i get  my app to refresh the pages from the local directory.

**Comment by engineering87** [comment]
Ensure that your nuget.config file is correctly configured to prioritize the local directory:

```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <add key="LocalPackages" value="C:\localpath\packages" />
    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
  </packageSources>
</configuration>
```

Then try `dotnet restore` and `dotnet watch run`.
